components/speedreader/speedreader_util.h

```diff
@@ -58,6 +58,12 @@ enum class DistillState {
 // Page is in reader mode or speedreader mode.
 bool PageStateIsDistilled(DistillState state);
 
+// Page can't be distilled.
+bool PageDoesntSupportDistillation(DistillState state);
```

Comment:
it's not really clear what method of these two to use and when. It sounds like `! PageSupportsDistillation()` is the same as `PageDoesn'tSupportDistillation()` but from the code this is not true. Can we switch to a single method or pick better names?